### PR TITLE
Add labels for istio

### PR DESCRIPTION
**Pull request by nveenjain**
Signed-off-by: Naveen Jain <nveenjain@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: smi-conformance/grpc/handlers.go [hunk 1]
```diff
@@ -21,20 +21,30 @@ var (
...
+		PortSvcC: "9091",
+	}
 )
 
 func (s *Service) RunTest(ctx context.Context, req *conformance.Request) (*conformance.Response, error) {
 	results := make([]*conformance.SingleTestResult, 0)
 	var config test_gen.ServiceMesh
 
+	config = linkerdConfig
+	labels := make(map[string]string)
```
Comment (nveenjain) [inline smi-conformance/grpc/handlers.go:36]: @kumarabd , can we accept labels from req, i wasn't sure of that, so hardcoded atm...
Only this argument needs to be changed for future changes too...
Rest should work fine...

Comment (kumarabd) [reply smi-conformance/grpc/handlers.go:36]: @nveenjain  sure let me have a look.

File: smi-conformance/test-gen/test_gen.go [hunk 2]
```diff
@@ -29,7 +29,14 @@ type Results struct {
 	} `json:"testcase"`
 }
 
+<<<<<<< HEAD
```
Comment (leecalcote) [inline smi-conformance/test-gen/test_gen.go:32]: @nveenjain I think we have an unresolved merge conflict here.

**Comment by kumarabd** [comment]
@nveenjain @leecalcote  Fixed everything. Its working now.

**Comment by leecalcote** [comment]
Thank you, @nveenjain @kumarabd

**Pull request merged**